pom.xml

```diff
@@ -17,6 +17,7 @@
         <additionalOptions>-Xdoclint:none</additionalOptions>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <protobuf.version>3.11.1</protobuf.version>
+        <log4j.version>2.15.0</log4j.version>
```

Comment:
Adding the property is not enough with this POM setup. Maybe because the project does not inherit from the spring starter parent, but includes it like a BOM. But no, just adding your suggestion doesn't fix it (already tried that one, too).

Reply:
Yeah, if you pull it in as a transitive dependency it will grab whatever. I tried the pom in this PR and its including 2.15.0 in the jar file. I would vote for getting it fixed ASAP and re working it for an elegant solution next.

Reply:
The log4j version should be bumped to 2.16, 2.15 is now considered an incomplete solution.

Reply:
```suggestion
        <log4j.version>2.16.0</log4j.version>
```

Reply:
@mfinger-incontact  @srinivassegu  Thank you for suggestion. Updated to 2.16